vmdk_plugin/drivers/vmdk/vmdk_driver.go

```diff
@@ -77,7 +79,23 @@ func NewVolumeDriver(port int, useMockEsx bool, mountDir string, driverName stri
 		}
 	}
 
-	d.refCounts.Init(d, mountDir, driverName)
+	refCountStatus := d.refCounts.Init(d, mountDir, driverName)
+	// If refcounting wasn't successfull, schedule one again
+	if refCountStatus == false {
+		timer := time.NewTimer(refCountDelay)
+		log.Infof("Refcounting failed. Scheduling again after %s seconds", refCountDelay)
+		go func() {
+			<-timer.C
+			refCountStatus = d.refCounts.Init(d, mountDir, driverName)
+			if refCountStatus {
+				log.Infof("Refcounting reattempt succeded")
+			} else {
+				log.Infof("Refcounting reattempt failed")
```

Comment:
This can be done. But I am not sure if is really required.
But the first refcount attempt might timeout only when both docker and plugin are being are coming up. In such cases, we retry after sufficient time cushion, taking into consideration the docker backoff time, ensuring it has come up. Can you elaborate on why the docker  might not reply to the api calls the second time?